src/main/java/it/gov/pagopa/payment/controller/payment/IdPayCodePaymentController.java

```diff
@@ -1,24 +1,27 @@
 package it.gov.pagopa.payment.controller.payment;
 
 import it.gov.pagopa.payment.dto.AuthPaymentDTO;
+import it.gov.pagopa.payment.dto.PinBlockDTO;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 @RequestMapping("/idpay/payment/idpay-code")
 public interface IdPayCodePaymentController {
 
-    @PutMapping("/{trxId}/relate-user") //TODO after refactor path
+    @PutMapping("/{transactionId}/relate-user") //TODO after refactor path
     @ResponseStatus(code = HttpStatus.OK)
     AuthPaymentDTO relateUser(@PathVariable("trxId") String trxId,
                               @RequestHeader("x-user-id") String userId);
 
-    @PutMapping("/{trxId}/preview") //TODO after refactor path
+    @PutMapping("/{transactionId}/preview") //TODO after refactor path
     @ResponseStatus(code = HttpStatus.OK)
     AuthPaymentDTO previewPayment(@PathVariable("trxId") String trxId,
                               @RequestHeader("x-user-id") String userId);
 
-    @PutMapping("/{trxId}/authorize") //TODO after refactor path
+    @PutMapping("/{transactionId}/authorize")
     @ResponseStatus(code = HttpStatus.OK)
-    AuthPaymentDTO authPayment(@PathVariable("trxId") String trxId,
-                               @RequestHeader("x-user-id") String userId);
+    AuthPaymentDTO authPayment(@PathVariable("transactionId") String trxId,
+                               @RequestHeader("x-acquirer-id") String acquirerId,
+                               @RequestHeader("x-merchant-fiscal-code") String merchantFiscalCode,
```

Comment:
APIm policy should resolve x-acquirer-id and x-merchant-fiscal-code into x-merchant-id
I don't expect that you are required to use these 2 values, but directly the merchantId